com.unity.formats.fbx/Editor/ConvertToNestedPrefab.cs

```diff
@@ -341,7 +346,11 @@ private static System.Reflection.MethodInfo SetSearchFilterMethod
         /// <returns></returns>
         internal static List<GameObject> GetSceneReferencesToObject(Object obj)
         {
-            var instanceID = obj.GetInstanceID();
+#if UNITY_6000_2_OR_NEWER
```

Comment:
Nit picking: fix is not needed as embedded in a `#if !UNITY_2021_2_OR_NEWER` = this will never be run in 6000.2.